packages/styles/src/menu-dropdown/menu-dropdown.scss

```diff
@@ -0,0 +1,95 @@
+@import "../core/_dev";
+
+.as-menu-dropdown {
+  --as-dropdown-menu-hover-color: #D3F6E6;
+  position: relative;
+
+  &--input {
+    width: 100%;
+    min-height: 32px;
+    margin: 0;
+    padding: get-spacing(2);
+    border: 0;
+    background: var(--as-color-ui-01, #{$color-ui-01});
+    color: var(--as-color-primary, #{$color-primary});
+    font: $font-body;
+    text-align: left;
+    cursor: pointer;
+
+    &:hover,
+    &:focus {
+      // Pending focus
+    }
+  }
+
+  &--options {
+    position: absolute;
+    z-index: 3;
+    width: 100%;
+    height: 0;
+    margin: 0 auto;
+    padding: 0;
+    overflow: hidden;
+    border-radius: 4px;
+    box-shadow: $shadow-8;
+
+    li {
+      display: flex;
+      align-items: center;
+      width: 100%;
+      transition: background-color 0.3s;
+      border: 0;
+      border-bottom: 1px solid var(--as-color-ui-04, #{$color-ui-04});
+      outline: none;
+      background: var(--as-color-ui-01, #{$color-ui-01});
+      color: var(--as-color-type-01, #{$color-type-01});
+      font: $font-body;
+      list-style: none;
+      cursor: pointer;
+
+      &:first-child {
+        border-top: 1px solid #F0F0F0;
+        border-radius: 4px 4px 0 0;
+      }
+
+      &:last-child {
+        border-radius: 0 0 4px 4px;
+      }
+
+      &:hover,
+      &:focus-within {
+        background: var(--as-dropdown-menu-hover-color, #D3F6E6);
+      }
+
+      button {
+        width: 100%;
+        height: 100%;
+        min-height: 32px;
```

Comment:
I don't know if we're using spacing units for height in other components, but we should, right?

Reply:
what do you mean?

Reply:
I mean that I don't know if we should put `min-height: get-spacing(8)`.  I don't think if we have ever used that on height/width, but I don't know if we should.